static/test/javascripts/spec/common/onward/breaking-news.spec.js

```diff
@@ -56,9 +57,9 @@ define([
                         return breakingNews();
                     }).then(function(result) {
                         // make sure the DOM has finished updating
-                        requestAnimationFrame(function () {
+                        setTimeout(function () {
```

Comment:
yup, the source code is doing a `setTimeout(() => promise.write(), 0)`, so the test has to wait for more than one animation frame